src/governance-classes.h

```diff
@@ -178,6 +178,7 @@ class CSuperblock : public CGovernanceObject
 
     static bool IsValidBlockHeight(int nBlockHeight);
     static CAmount GetPaymentsLimit(int nBlockHeight);
+    static void GetSuperblockHeight(int& nBlockHeight, int& nLastSuperblock, int& nNextSuperblock);
```

Comment:
`nBlockHeight` should not be returned and imo it makes little sense to pass it via `&` since it's a simple type.